pkg/operator/ceph/cluster/utils.go

```diff
@@ -32,26 +32,61 @@ import (
 // Its content allows modifying Ceph configuration flags
 func populateConfigOverrideConfigMap(clusterdContext *clusterd.Context, namespace string, ownerInfo *k8sutil.OwnerInfo) error {
 	ctx := context.TODO()
-	placeholderConfig := map[string]string{
-		k8sutil.ConfigOverrideVal: "",
+	// Add the helm labels in case the user wants to install the cluster helm chart to start managing it
+	labels := map[string]string{
+		"app.kubernetes.io/managed-by": "Helm",
 	}
-
-	cm := &v1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      k8sutil.ConfigOverrideName,
-			Namespace: namespace,
-		},
-		Data: placeholderConfig,
+	// Add the helm annotations in case the user wants to install the cluster helm chart to start managing it
+	annotations := map[string]string{
```

Comment:
this would be an optional thing, needed to add it at Cephcluster CR?

Reply:
We just need it on this one configmap for this scenario.